Makefile

```diff
@@ -101,6 +101,7 @@ test/unit: $(GINKGO)
 test/e2e: $(OPERATOR_SDK)
 	# We have to unset mod=vendor here since operator-sdk is already
 	# building with it, and go tool fail if it's specified twice
+	mkdir -p test_logs/e2e
```

Comment:
On CI, we should copy this folder to exported artifacts, so it is available in the prow build. See https://github.com/nmstate/kubernetes-nmstate/blob/master/automation/check-patch.setup.sh#L12